benchmarks/main.yml

```diff
@@ -0,0 +1,437 @@
+name: spring-quarkus-perf-comparison
+states:
+  env.USER: someuser
+  env.HOST: 127.0.0.1
+  env.TARGET: ${{env.USER}}@${{env.HOST}}
+
+  config.JAVA_VERSION: 25-tem
+  config.GRAALVM_VERSION: 25-graalce
+
+  config.QUARKUS_VERSION: #3.28.3
+  config.SPRING_BOOT_VERSION: #3.5.6
+
+  config.JVM_MEMORY: #-Xmx128m
+  config.CMD_PREFIX: #taskset --cpu-list 0-3
+
+  config.NATIVE_QUARKUS_BUILD_OPTIONS: #-Dquarkus.native.native-image-xmx=<maximum_memory>
+  config.NATIVE_SPRING_BUILD_OPTIONS:
+
+  config.cgroup.cpu: "0,2,4,6,8,10,12,14"
+  config.cgroup.mem_max: "14G"
+  config.cgroup.name: spring-quarkus-perf-comparison
+
+  config.profiler.name: none #jfr flamegraph
+  config.profiler.events: cpu
+
+  config.repo.branch: main
+  config.repo.url: https://github.com/quarkusio/spring-quarkus-perf-comparison.git
+
+  RUN.WRK_BIN: jbang wrk@hyperfoil
```

Comment:
This is what is reported by `wrk`  while extracting manually the report:

[wrk.html](https://github.com/user-attachments/files/22995209/wrk.html)

Some notes:
1. calibration phase duration is hardcoded in the command (which is not great)
2. the report is pretty boring per se: latencies are useless (due to coordinated omission) and, as expected, throughput is stable

Which makes the report per se not as interesting as I was hoping (due to the simplicity of the workload...)
In short: I don't think we need to extract the report if we want to keep this test as simple as this.

Reply:
@edeandrea PTAL